### PR TITLE
get projection from .projection file first #patch

### PR DESCRIPTION
**Pull request by ar-siddiqui**
This PR:
- Change getting projection logic by first trying to get exactly from model.projection file and then if that doesn't exist then try to get from other .prj or .projection files.
- Improve error messages.

### CODE REVIEW

File: tools/model.go [hunk 1]
```diff
@@ -282,14 +282,14 @@ func getProjection(rm *RasModel, fn string, wg *sync.WaitGroup) {
 	sourceSpRef := gdal.CreateSpatialReference(line)
 	if err := sourceSpRef.Validate(); err != nil {
 		if filepath.Ext(fn) == ".prj" {
-			fmt.Println(fmt.Sprintf("%s is not a valid projection file, check that there are not multiple project files.\n", fn))
+			fmt.Println(fmt.Sprintf("%s is not a valid projection file.\n", fn))
```
Comment (ShaneMPutnam) [inline tools/model.go]: The previous message and logic was designed to tell the user that there may be a second model .prj file in the same directory as the primary model .prj. If you really don't want this extra hint, then there is no reason to check the file extension and have this if else statement. Instead you should just print either the original validation error or your custom message.

File: tools/model.go [hunk 2]
```diff
@@ -337,8 +342,8 @@ func NewRasModel(key string, fs filestore.FileStore) (*RasModel, error) {
 			rasWG.Flow.Add(1)
 			go getFlowData(&rm, fp, &rasWG.Flow)
 
-		case rasRE.Projection.MatchString(ext):
-			if filepath.Base(key) != filepath.Base(fp) {
+		case rm.Metadata.Projection == "" && rasRE.Projection.MatchString(ext):
```
Comment (ShaneMPutnam) [inline tools/model.go:342]: What if there are two projection files, whether those are .prj or .projection in the same directory as the model .prj file, you don't want the user to know? I didn't think this actually caused an error, just printed the error.

**Review by ShaneMPutnam — CHANGES_REQUESTED** [review]
Please update the code as suggested or reply explaining why you think it is okay as it is.

File: tools/model.go (hunk 1 above)
Comment (ar-siddiqui) [reply tools/model.go]: Revised code.

File: tools/model.go (hunk 2 above)
Comment (ar-siddiqui) [reply tools/model.go:342]: Discussed and explained.

**Review by ShaneMPutnam — APPROVED** [review]
Thanks, looks good!

**Pull request merged**